SolidStateGroup/bullet-train-api
# fix: UI issue when there were more than 100 features

Thanks for submitting a PR! Please check the boxes below:


## Changes

- Install and use react-virtualized inside react-window

## How did you test this code?

- Go to Features page.
- Create more than 100 features.
- Click any feature.
- Go to "Segment Overrides" tab.
- Click "Update Segment override".
- The error no longer occurs and works correctly.
